xds/internal/balancer/clusterresolver/config.go

```diff
@@ -168,18 +168,13 @@ type LBConfig struct {
 	XDSLBPolicy *internalserviceconfig.BalancerConfig `json:"xdsLbPolicy,omitempty"`
 }
 
-const (
-	rrName = roundrobin.Name
-	rhName = ringhash.Name
```

Comment:
Personally, I don't mind the alias since it saves verbosity, but I'm indifferent.

Reply:
`ringhash.Name` is not very long compared to `rhName` but it is way more descriptive.